### DISCUSSION TITLE
v1.6

### DISCUSSION THREAD

**Discussion by J535D165** [asreview/asreview#1706] [Show and tell]
## What's Changed
* Change default production WSGI server by waitress and add support for other WSGI servers by @J535D165 in https://github.com/asreview/asreview/pull/1696
* Backport info banner at the login screen to version 1.6 by @J535D165 in https://github.com/asreview/asreview/pull/1698
* Add support for customizable API URL in production without rebuilding by @J535D165 in https://github.com/asreview/asreview/pull/1699
* Improve password manager and autocomplete features for account creation by @J535D165 in https://github.com/asreview/asreview/pull/1702
* Improve Dockerfile file and optimize for server stack use @J535D165 in https://github.com/asreview/asreview/pull/1703

## Important info for server deployment 

❗ This version of ASReview LAB improves server and flask-related code. Users running ASReview in production with config TOML files and environment variables should take care of unwanted changes and test their applications. We tried to make everything backward compatible for server users, but unexpected behavior is still possible. It is important to change environment variables to a variant with the `ASREVIEW_LAB_` prefix. See PR https://github.com/asreview/asreview/pull/1696 for most of the relevant changes. 

ℹ️ Server users: please check out https://github.com/asreview/asreview-server-stack. This is our ASReview LAB server stack with reverse proxy, robust Gunicorn server, and PostgreSQL database.  

**Full Changelog**: https://github.com/asreview/asreview/compare/v1.5.1...v1.6

<hr /><em>This discussion was created from the release <a href='https://github.com/asreview/asreview/releases/tag/v1.6'>v1.6</a>.</em>

**Comment by SamyAteia** [comment]
Is it intended that the development environment now starts with authentication enabled per default?
